lib/codemirror.dart

```diff
@@ -1124,7 +1124,7 @@ class TextMarker extends ProxyHolder {
     if (result is! JsObject) return null;
 
     try {
-      if (result is Map) {
+      if (result is JsObject && result.hasProperty('from')) {
```

Comment:
is the '`result is JsObject`' needed to satisfy the analyzer? We're already doing an `is!` check above

Reply:
Interesting. It is needed to get the analyzer to suggest the right completions on result but the analyzer doesn't seem to complain without it.